components/payments/internal/storage/migrations_v1.x.go

```diff
@@ -179,6 +179,41 @@ func registerMigrationsV1(ctx context.Context, migrator *migrations.Migrator) {
 					return err
 				}
 
+				return nil
+			},
+		},
+		migrations.Migration{
+			Up: func(tx bun.Tx) error {
+				_, err := tx.Exec(`
+					CREATE TABLE IF NOT EXISTS transfers.transfer_initiation_adjustments (
+						id uuid NOT NULL,
+						transfer_initiation_id CHARACTER VARYING NOT NULL,
+						created_at timestamp with time zone  NOT NULL DEFAULT NOW() CHECK (created_at<=NOW()),
+						status int NOT NULL,
+						error text,
+						metadata jsonb,
+						CONSTRAINT transfer_initiation_adjustments_pk PRIMARY KEY (id, transfer_initiation_id)
```

Comment:
id being a uuid, I don't think you have benefits to create a compound primary key

Reply:
Done thanks !